samples/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@google-cloud/pubsub": "*",
-    "@google-cloud/storage": "^2.0.3",
+    "@google-cloud/storage": "+git",
```

Comment:
@stephenplusplus, not sure what to do here. Should I revert or change this before merging?

Reply:
You can just keep this the same as it was. When we cut a release, the samples/package.json dependency version is updated automatically.

To run the sample tests locally, `npm run samples-test` from the root directory of the repo should handle the linking automatically!